script.py

```diff
@@ -0,0 +1,60 @@
+import copy
+import json
+from pprint import pprint
+
+from great_expectations._docs_decorators import _public_api_registry
+
+
+def find_classes_that_need_public_api_decorator() -> list[str]:
+    classes = []
+    for class_, methods in _public_api_registry.items():
+        if "__init__" not in methods:
+            classes.append(class_)
+
+    return sorted(classes)
+
+
+def write_public_api_registry_file() -> None:
+    registry = copy.deepcopy(_public_api_registry)
+    for k, v in registry.items():
+        registry[k] = sorted(v)
+
+    with open("public_api_registry.json", "w") as f:
+        json.dump(registry, f, indent=2)
+
+
+if __name__ == "__main__":
+    classes = find_classes_that_need_public_api_decorator()
+    pprint(classes)
+    """
+    great_expectations.checkpoint.checkpoint.CheckpointResult
```

Comment:
why this list of classes?

Reply:
These are classes that have `public_api` methods without actually being decorated themselves (resulting in them not showing up in docs)